medusa/server/api/v2/episode.py

```diff
@@ -55,6 +55,8 @@ def get(self, series_slug, episode_slug, path_param):
         if path_param:
             if path_param == 'metadata':
```

Comment:
Should the releases not be sent back with the metadata? Since it's coming from the cache I can't see any performance downside to this.